ci-operator/config/cri-o/cri-o/cri-o-cri-o-main__ci.yaml

```diff
@@ -28,9 +28,13 @@ images:
 - dockerfile_literal: |
     FROM base
     RUN dnf install -y jq git openssl python3.11 google-cloud-cli-447.0.0-1 gzip jq unzip openssh-clients openssl bind-utils util-linux xz && \
```

Comment:
xref: https://stackoverflow.com/questions/73263452/how-to-install-google-cloud-sdk-on-rhel-9
per this post, it should be `google-cloud-cli`

Reply:
https://github.com/openshift/installer/blob/f60ebb065b4242586f7afacc5f2be8afddbdfbde/images/installer/Dockerfile.upi.ci#L25C1-L25C80